specification/httpxe/src/main/scripts/org/kaazing/specification/httpxe/responses/wrapped.501.response.in.200/request.rpt

```diff
@@ -19,12 +19,17 @@ connected
 
 write method "GET"
 write version "HTTP/1.1"
-write flush
+write header host
+write header "Content-Type" "application/x-message-http"
+write header "X-Next-Protocol" "httpxe/1.1"
+write header content-length
+write "GET /path HTTP/1.1\r\n"
+      "Content-Length: 0\r\n"
+      "\r\n"
+write close
 
 read status "200" "OK"

```

Comment:
Specification says 5xx status codes should be in enveloped response. Here it is 200. Should this be changed ?